test/views/ReportingFrequencyViewSpec.scala

```diff
@@ -89,651 +127,538 @@ class ReportingFrequencyViewSpec extends TestSupport {
 
   "ReportingFrequencyView" when {
 
-    "optInOptOutContentUpdateR17 is true" when {
-
-      "the user is an Agent" should {
+    "the user is an Agent" should {
 
```

Comment:
for these tests can we split it out into when switch is enabled and when it is not enabled 

Like this:

<img width="1219" alt="Screenshot 2025-07-10 at 10 10 28" src="https://github.com/user-attachments/assets/2e451cd0-5fb0-48f8-a50c-ec9da92fa5aa" />

Reply:
will be easier for clean up later on